src/main/java/seedu/address/logic/commands/MarkAttendanceCommand.java

```diff
@@ -50,6 +50,7 @@ public CommandResult execute(Model model) throws CommandException {
         Person markedPerson = new Person(
                 personToMarkAttendance.getName(), personToMarkAttendance.getPhone(),
                 personToMarkAttendance.getEmail(), personToMarkAttendance.getAddress(),
+                personToMarkAttendance.getPayment(),
```

Comment:
Should this be creating a new payment object set to true instead since its the person to be marked?